src/components/Home.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Header from './Header'
+import Product from './Product'
+
+const Home = () => {
+  return (
+    <>
+    <div>
+      <div>
+        <Header/>
+        <div className='m-4 mt-8 p-2 bg-blue-100 w-fit rounded-md'>
+            Hello, This project will show my development skills in React throughout this 30 day journey.
+        </div>
+      </div>
+      <div className='flex space-x-4 mx-4 my-8 justify-center'>
+        <Product image='Iphone.jpg' title="Iphone" description=" Available on Amazon" />
```

Comment:
Make a product_info.js and iterate over list of objects instead of hardcoding like this